elasticsearch/src/main/java/org/hibernate/search/elasticsearch/client/impl/JestClient.java

```diff
@@ -119,9 +119,23 @@ public void start(Properties properties, BuildContext context) {
 						),
 						TimeUnit.SECONDS
 				)
-				.gson( gsonService.getGson() )
-				.build()
+				.gson( gsonService.getGson() );
+
+		String username = ConfigurationParseHelper.getString(
+				properties,
+				CLIENT_PROP_PREFIX + ElasticsearchEnvironment.SERVER_USERNAME,
+				null
 		);
+		if ( !StringHelper.isEmpty( username ) ) {
```

Comment:
so a zero-length string would disable it? Should we rather check for not-null, or is that going to be all the same for ES?